src/import/import-compress.c

```diff
@@ -293,19 +294,28 @@ int import_compress_init(ImportCompress *c, ImportCompressType t) {
 #if HAVE_ZSTD
         case IMPORT_COMPRESS_ZSTD: {
                 size_t r2;
+                int level = ZSTD_CLEVEL_DEFAULT;
```

Comment:
I think the approach with the variable is acceptable. But please make the default something reasonable, whatever you think makes sense.

Also, the new variable needs to be (briefly) described in `docs/ENVIRONMENT.md`.

Reply:
I have documented the variable (I'll pick a better default once I do some tests), but see my comment below.